### PR TITLE
Log ert shared main

### PR DESCRIPTION
**Pull request by dafeda**
**Approach**
Moving logging statement to after handler has been added such that logs are sent to Azure.
A bit of cleanup as well.

### CODE REVIEW

File: ert_shared/main.py [hunk 1]
```diff
@@ -413,10 +414,10 @@ def main():
     if args.verbose:
         logger.setLevel("DEBUG")
     FeatureToggling.update_from_args(args)
-    logger.info("Running ert with {}".format(str(args)))
     try:
         with start_ert_server(), ErtPluginContext() as context:
             context.plugin_manager.add_logging_handle_to_root(logging.getLogger())
+            logger.info(f"Running ert with {str(args)}")
```
Comment (oyvindeide) [inline ert_shared/main.py:420]: Agree with other changes, but this should stay as `.format` (logging is the exception to the rule)

Comment (dafeda) [reply ert_shared/main.py:420]: The only recommendation I've seen is to use `%s`-formatting: 
https://docs.python.org/3/howto/logging.html#logging-variable-data

Why should we prefer `.format` to f-strings?

Comment (oyvindeide) [reply ert_shared/main.py:420]: Was thinking of pylint W1202, but you are right, that only recommends `%s` formatting, and should be no difference between `f""` and `.format()`.

**Comment by oyvindeide** [comment]
Jenkins: test this please

**Review by oyvindeide — APPROVED** [review]

**Comment by oyvindeide** [comment]
The error seems to be unrelated: 
```
148/148 Test  #88: row_scaling ...............................Subprocess terminated***Exception:  66.12 sec


99% tests passed, 1 tests failed out of 148

Label Time Summary:
EquinorData    =  39.84 sec*proc (31 tests)

Total Test time (real) =  66.41 sec

The following tests FAILED:
	 88 - row_scaling (Subprocess terminated)
Errors while running CTest
script returned exit code 8
```
but can you see if you can replicate locally?

**Comment by dafeda** [comment]
Jenkins test this please

**Comment by dafeda** [comment]
> The error seems to be unrelated:
> 
> ```
> 148/148 Test  #88: row_scaling ...............................Subprocess terminated***Exception:  66.12 sec
> 
> 
> 99% tests passed, 1 tests failed out of 148
> 
> Label Time Summary:
> EquinorData    =  39.84 sec*proc (31 tests)
> 
> Total Test time (real) =  66.41 sec
> 
> The following tests FAILED:
> 	 88 - row_scaling (Subprocess terminated)
> Errors while running CTest
> script returned exit code 8
> ```
> 
> but can you see if you can replicate locally?

I ran the tests locally, and of all of them run fine except `tests/libres_tests/res/util/test_subprocess.py` which hangs.
It hangs on `main` as well, so it is not specific to this branch.

**Comment by sondreso** [comment]
Jenkins test this please

**Comment by sondreso** [comment]
Sorry I aborted the wrong test :disappointed:

**Comment by oyvindeide** [comment]
The failing test is not required, but it is a bit worrying that it keeps failing - that being said, I really cant see how it should be related to your changes.

**Comment by oyvindeide** [comment]
> 
> I ran the tests locally, and of all of them run fine except `tests/libres_tests/res/util/test_subprocess.py` which hangs.
> It hangs on `main` as well, so it is not specific to this branch.

This was a C-test, did you run those as well?

**Comment by oyvindeide** [comment]
jenkins: test this please

**Pull request closed without merging**